fourmolu.yaml

```diff
@@ -0,0 +1,10 @@
+indentation: 2
+function-arrows: leading
+comma-style: leading
```

Comment:
I also don't reccomen this, it causes worse git diffs usually.

Reply:
I tried using it because of its better "git diff"-ability. Can you elaborate?

Reply:
Actually my original reasoning was wrong/not always right, since the ormolu default isn't any better. But I usually do it for consistency everywhere, because there's some logic behind trailing arrows, and also some things are more git-friendly with trailing commas:
```haskell
import M
  (
    a,
    b,
    c,
    d,
  )
```
is better than
```haskell
import M
  ( a
  , b
  , c
  , d
  )
```
Because removing the `a` causes a bigger diff than the rest.

Unfortunately trailing commas aren't supported in lists or records yet, so it's not as helpful there.